src/huggingface_hub/hf_api.py

```diff
@@ -1533,8 +1563,10 @@ def delete_file(
 
         r.raise_for_status()
 
+    @_deprecate_positional_args
     def get_full_repo_name(
         self,
+        *,
         model_id: str,
         organization: Optional[str] = None,
         token: Optional[str] = None,
```

Comment:
On the fence about this one too, wdyt @osanseviero?

Reply:
No strong opinion, but I think `get_full_repo_name("repo_name")` is intuitive, and `get_full_repo_name(model_id="repo_name")` might be a bit redundant